### ISSUE TITLE
video output black 

### ISSUE THREAD

**Issue by Colliwomple** [volotat/DiffMorph#7]
Latest try:

python morph.py -s images/img_1.jpg -t images/img_2.jpg

2021-01-12 17:23:25.848313: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN)to use the following CPU instructions in performance-critical operations: SSE4.1 SSE4.2 AVX AVX2 FMA

To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.

2021-01-12 17:23:25.869795: I tensorflow/core/platform/profile_utils/cpu_utils.cc:104] CPU Frequency: 3600000000 Hz

2021-01-12 17:23:25.870428: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x557665775f10 initialized for platform Host (this does not guarantee that XLA will be used). Devices:

2021-01-12 17:23:25.870473: I tensorflow/compiler/xla/service/service.cc:176] StreamExecutor device (0): Host, Default Version

Epoch 100, Loss: 0.07585608214139938

Epoch 200, Loss: 0.056177154183387756

Epoch 300, Loss: 0.047623421996831894

Epoch 400, Loss: 0.04261484742164612

Epoch 500, Loss: 0.03922488912940025

Epoch 600, Loss: 0.036724839359521866

Epoch 700, Loss: 0.03478630632162094

Epoch 800, Loss: 0.03322023153305054

Epoch 900, Loss: 0.031917549669742584

Epoch 1000, Loss: 0.030810609459877014

Image #10 produced.

Image #20 produced.

Image #30 produced.

Image #40 produced.

Image #50 produced.

Image #60 produced.

Image #70 produced.

Image #80 produced.

Image #90 produced.

Image #100 produced.

Result video saved.

So far it looks promising but in the morph folder the video is just 630 kb and black when trying to play. 2 pictures maps.jpg +result.jpg look better see https://i.imgur.com/BbhNnsZ.jpg and https://i.imgur.com/6pYS0gw.jpg . Seems like i am coming closer but have no idea why the video is not created

**Comment by volotat** [comment]
Sorry, but I have no idea what your problem might be. One thing I could suggest is to check out if you have mp4v codec installed on your system.
